spot-ml/INSTALL.md

```diff
@@ -14,6 +14,7 @@ Names and language that we will use from the configuration variables for Spot (t
 - MLNODE The node from which the spot-ml routines are invoked
```

Comment:
maybe this is a ticket for future work... but we have the opportunity to greatly simplify ML installation and execution (and rid ourselves of the concept of an ML node)